### ISSUE TITLE
Edit buttons on YouTube videos appear incorrectly.

### ISSUE THREAD

**Issue by DamienDavisNeff** [DamienDavisNeff/RedSubscribeButton#14]
If you are the owner of a YouTube video, the edit buttons behave incorrectly.

The *Edit Video* button appears red (like the normal subscribe button), which can be fixed by checking the [aria-label~="Subscribe"].

The *Analytics* button and *Edit Video* button do not have the boxy shape when you have that option enabled. This can be fixed by specifically adding both of those to the css file.

![image](https://user-images.githubusercontent.com/105996288/224556279-1222add6-b50a-4dbb-97fc-29b655756a6c.png)

**Comment by DamienDavisNeff** [comment]
This has been fixed, and will release alongside the next update. 

![image](https://user-images.githubusercontent.com/105996288/224556338-e700789c-a85a-4267-be94-db09f64fa185.png)

While I was at it, I also applied the other color scheme to it.
![image](https://user-images.githubusercontent.com/105996288/224556768-1c1b9d6c-3bd9-4207-9b42-ecd2a5e61696.png)

**Comment by DamienDavisNeff** [comment]
This bug has reappeared, after the classes for the button has changed. 

I'll fix it alongside #25 & #24 when I get a chance to.

**Comment by DamienDavisNeff** [comment]
Actually I don't think it's this specific bug, but it's similar enough to be apart of the same report, however.

**Comment by DamienDavisNeff** [comment]
This is an updated screenshot
![image](https://github.com/DamienDavisNeff/RedSubscribeButton/assets/105996288/ff41f8d4-afa6-4d78-b78c-308f3112ad07)
![image](https://github.com/DamienDavisNeff/RedSubscribeButton/assets/105996288/b47d8ec3-b176-401e-97dd-c2ebb168b0af)
This also impacts playlists. 
![image](https://github.com/DamienDavisNeff/RedSubscribeButton/assets/105996288/088be179-37b5-42bf-b794-98ec5b8865e9)

**Comment by DamienDavisNeff** [comment]
This was caused by my fix for #23
```css
a:not(#video-title)
{
    background-color:rgba(0,0,0,0) !important;
}
```
Since the aria-labels for "Edit" & "Analytics" are no longer present, it caused them to fall under that category.

**Comment by DamienDavisNeff** [comment]
While that fix does fix the transparency issue, it does not return the color to all of those elements, but since those are misc elements that aren't seen often, I don't really think it's an issue, and so I'm going to close this.
